lib/MusicBrainz/Server/Entity/LinkAttributeType.pm

```diff
@@ -75,6 +75,8 @@ around TO_JSON => sub {
         $self->link_entity('link_attribute_type', $root->id, $root);
     }
 
+    my @children = map {$_->TO_JSON} @{$self->children};
```

Comment:
(Nitpick: there's an `all_children` accessor to get a list without dereferencing, and we typically put spaces inside inline code blocks.)
```suggestion
    my @children = map { $_->TO_JSON } $self->all_children;
```

Reply:
Actually, would we also be able to just use `$self->sorted_children` here and remove the `compareChildren` function in JS?

Reply:
sorted_children needs a collator - is there a way to get a collator from Entity? (IIRC I can't just use $c)

Reply:
Not currently at least, so this ought to be fine.